### PR TITLE
WEBUI-44: enqueue debounce to prevent request cancelling 10.10

### PR DESCRIPTION
**Pull request by richardsd**
Pipeline running https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/521/
This is the backport for WEBUI-44 in master.

### CODE REVIEW

File: elements/nuxeo-results/nuxeo-results.html [hunk 1]
```diff
@@ -377,15 +379,18 @@
       },
 
       fetch: function() {
-        return new Promise(function(resolve, error) {
-          this.debounce('fetch', function() {
+        return new Promise((resolve, error) => {
+          this._fetchDebouncer = Polymer.Debouncer.debounce(this._fetchDebouncer, Polymer.Async.timeOut.after(100), () => {
```
Comment (semisse) [inline elements/nuxeo-results/nuxeo-results.html:383]: On 10.10, to keep consistency, we should avoid ES6 syntax.

```suggestion
          this._fetchDebouncer = Polymer.Debouncer.debounce(this._fetchDebouncer, Polymer.Async.timeOut.after(100), function() => {
```

Comment (richardsd) [reply elements/nuxeo-results/nuxeo-results.html]: Thank you 👍

**Comment by semisse** [comment]
I know our pipelines are somewhat unstable, but did you run them?

**Comment by richardsd** [comment]
> Pipeline running [qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/521](https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/521/)
> This is the backport for WEBUI-44 in master.

@semisse I did but it failed. I am relaunching it again.
https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/522/

**Review by mnixo — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**